ckanext/dcatapit/templates/snippets/dcatapit_form_fields.html

```diff
@@ -11,7 +11,7 @@
                                  id='field-' + field.name,
                                  placeholder=field.placeholder,
                                  type=field.type,
-                                 value=data[field.name],
+                                 value=h.format((data[field.name] or ''), '%Y-%m-%d', field.type),
```

Comment:
@cezio 
why here the default format is not equal to '%d-%m-%Y' ?

Reply:
because it's edit form, and it requires this format to have widget populated correctly

Reply:
To give some more context on the issue: input value can be in different formats, so in `IPackageController.after_show` I'm normalizing it to format from field definition (in dcatapit's schema). This is however not final processing. In edit form, this value is passed to `<input type="date"../>` field, which is interpreted by browser in html5. This is why we had errors in dev console in https://github.com/geosolutions-it/ckanext-datitrentinoit/issues/35#issuecomment-426365662

`Input[type=date]` field requires values to be in `%Y-%m-%d` format, so I have to reformat those in the template. After that, they're used in widget provided by browser (works in modern browser, older browsers will display regular input field). The last element is the way date is displayed in this widget. This is controlled by browser's locale.
Here's some reference docs for that field: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date